test_libs/pyspec/eth2spec/test/helpers/custody.py

```diff
@@ -20,14 +20,14 @@ def get_valid_early_derived_secret_reveal(spec, state, epoch=None):
         ),
     )
     mask = bls_sign(
-        message_hash=spec.hash_tree_root(epoch),
+        message_hash=spec.hash_tree_root(spec.Epoch(epoch)),
         privkey=privkeys[masker_index],
         domain=spec.get_domain(
             state=state,
             domain_type=spec.DOMAIN_RANDAO,
             message_epoch=epoch,
         ),
-    )
+    )[:32]  # TODO(Carl): mask is 32 bytes, and signature is 96? Correct to slice the first 32 out?
```

Comment:
Is the point of this to be more MPC friendly than hashing the signature?

Reply:
See older status update and comment by Carl. It's good enough for now, but (as I also suggested to him), a hash may be used too. It's something for a later PR however, as it's mostly just a problem uncovered by the typing we have now.